packages/mdc-toolbar/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {MDCComponent} from '@material/base';
+
+import MDCToolbarFoundation from './foundation';
+import * as util from './util';
+
+export {MDCToolbarFoundation};
```

Comment:
we should probably `export {util}` here too (and document its API in the README), so that users can take advantage of `applyPassive()`. 

>side note: it's probably time to create a tech-debt issue to refactor some of this common util logic into a helper mdc library